script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -66,19 +66,19 @@ It's the curry bread sold at the famous bakery in front of the station, right?
 It's not 'why'.
 I don't know what to say to that 'Are you sure you're from Earth if this doesn't make you happy?' look she's giving me.
 <Page22>
-"You had the same thing yesterday. ...Senpai, do you like spicy food?"//can be phrased better
-For some reason, I avoid saying 'Do you like curry?'
+"You had the same thing yesterday... senpai, do you like spicy food?"//can be phrased better
+For some reason, I avoid specifically saying 'Do you like curry?' //Recheck on tl
```

Comment:
other than "なんとなく" being translated a little weird, I think the existing TL is basically fine (i also think the jp's tense is better since he already asked)
> Without knowing why, I specifically avoided asking 'do you like curry?'